answers.md

```diff
@@ -1 +1,18 @@
 # Answers Go In This File
+//Shaina 
+//Qustion 1
+1. Git is a tool used by software developers to manage their code. Git keeps track of code by saving existing code. Software developers use it to code with others by branching, merging commits, and adding their code to the main base. Git is a local repository.
+//Question 2
+2. GitHub is like a social media platform, or a version control system, for coders or software developers. It allows software developers to start repositories, fork other repositories, and request pulls. GitHub is a remote repository.
+//Question 3
+3. Commit is a separate branch of a repository. It allows developers to change or add to the main code without changing or affecting the original or main base. Editing code in a commit is different from editing a document or file in a way that if you make a change in a google document or file, it will affect the document or file. Everyone will see the change you made to the original document or file. Commit allows you to make changes you like without affecting the original repository. If you want someone to see your commit and make changes to the original repository, you can request a pull to merge your commit and add it to the original repository. 
```

Comment:
This answer doesn't accurately define commits — much of what you're describing sounds more like what a "branch" is. What we're looking for is a description of a commit as a "snapshot" of your repository.

I would recommend googling "What is a commit Github". You should find a bunch of resources but the one I recommend is the [definition of commits on the Github glossary](https://docs.github.com/en/get-started/quickstart/github-glossary). I would also take another look at the [lecture notes](https://github.com/The-Marcy-Lab-School/Fall-2022-Curriculum-BMC/blob/main/se-unit-0/lesson_1_git/git_lecture_notes.md#git--github-developing-a-mental-model) to develop a better understanding of what a commit is.